### ISSUE TITLE
Let us set chromecast volume

### ISSUE THREAD

**Issue by mahyar** [andresgottlieb/soundcast#15]
The chromecast device has a separate volume setting from your system volume, so it would be nice to be able to set it inside the menubar app.  Right now I have to go into the chromecast app to set the chromecast volume.

Thanks!

**Comment by andresgottlieb** [comment]
Hi @mahyar,

 Very good idea. I think I know how to make it happen. I'll let you know how it goes.

Thanks for the feedback!

**Comment by Robbertdv** [comment]
Hee!

Any updates on this subject? I'd like to set the volume also ;-)
I'm currently using the chromecast audio which has a different volume balance when used from my iPhone. When I run it on my mac it's around 25% of the maximum volume.

Kind regards,
Robbert

**Comment by simombo** [comment]
I second (or third) that!

This app saved my ass, when I realized Chromecast Audio isn't compatible with Macbooks.

**Comment by matthijsariens** [comment]
I agree! This is a great app, but with volume control it will be an awesome app!

**Comment by sheldoan** [comment]
+1

**Comment by Gundisalv** [comment]
+1!

**Comment by luatnd** [comment]
+1 !
My Mac volume is too low, just 25% - 30% compared to my iphone with AllCast app.

**Comment by golovlev** [comment]
+1 pls. Volume level from macbook is too low, i have to boost volume on my speakers to the max.

**Comment by Si-cox** [comment]
+1 Here - Fixing this would make the experience a HUGE amount better..

**Comment by jjaimon** [comment]
+1 Right now I've to boost volume on my audio player to max.
